e2e/setup/config.go

```diff
@@ -84,6 +88,10 @@ func NewConfig() (Config, error) {
 			Logger: logger,
 			SchemeBuilder: k8sclient.SchemeBuilder{
 				corev1alpha1.AddToScheme,
+				providerv1alpha1.AddToScheme,
```

Comment:
I don't think they were needed just yet, but I didn't want to hit this problem again in the future where some scheme is missing. That's why I'm adding them all now.